src/index.js

```diff
@@ -3,8 +3,8 @@
 import "./tailwind.css";
 require("./styles.scss");
 
-const {Elm} = require('./Main');
-var app = Elm.Main.init({flags: 6});
+const { Elm } = require('./elm/Main');
```

Comment:
what is new directory? this is not part of the repo

Reply:
I added a sub directory for the elm files in src.  this can be reverted if you don't want it.

Reply:
Yes, this is an Elm starter, so I don't to hide the elm code 2 directories down :-)